tensorboard/plugins/core/core_plugin.py

```diff
@@ -42,6 +43,11 @@
 JS_CACHE_EXPIRATION_IN_SECS = 86400
 
 
+class NotificationCategoryEnum(Enum):
```

Comment:
Minor, but since this class isn't used yet in this PR, mind waiting to introduce it until it's needed?